test/langtools/tools/javac/sealed/erroneous_permits/MultipleErroneousPermittedTypes.java

```diff
@@ -0,0 +1,7 @@
+/*
+ * @test /nodynamiccopyright/
```

Comment:
it would be preferable for this test to be moved to a subtest in `test/langtools/tools/javac/sealed/SealedCompilationTests.java`

Reply:
Moved it into `testTypeVarInPermits`. Please let me know if it is fine like that.

Reply:
looks good, just please remember to update the copyright year to 2025 before pushing